src/api/symbols.js

```diff
@@ -5,3 +5,5 @@ export const props = '____props';
 export const renderer = '____renderer';
 export const rendererDebounced = '____rendererDebounced';
 export const shadowRoot = '____shadowRoot';
+export const v0ShadowDOMProperty = 'createShadowRoot'; // property used to check for shadowDOMv0 support
```

Comment:
Eventually these symbols will actually use `Symbol()`, which you wouldn't want to do with these. I'd suggest just co-locating these in the support file.